### PR TITLE
Leaderboard hit api

### PR DESCRIPTION
**Pull request by principles31**
# Leaderboard: hit the API
- The goal of this project is to work on the Leaderboard list app, sending and receiving data from/to the Leaderboard API.
# Project requirements
- Read the [Leaderboard API documentation](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) to learn how to use this API.
- Create a new game with the name of your choice by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use async and await JavaScript features to consume the API.
- No styling is required.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+
+    //   if (!score.value.match(regex2)) {
+    //     errorMessage.style.display = "block";
+    //     errorMessage.style.color = "yellow";
+    //     errorMessage.style.fontSize = "12px";
+    //     errorMessage.style.lineHeight = "24px";
+    //     return;
+    //   }
+    //   errorMessage.style.display = "none";
+
```
Comment (Wes-Isaac) [inline src/index.js]: - [x] Great job, to keep your code clean, please don't leave commented code.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @principles31, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are set up properly ✔️.
- Great summary😲. 
- Readme looks professional 👀.
- Project works as expected 😮

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.js (hunk 1 above)
Comment (principles31) [reply src/index.js]: @Wes-Isaac Thanks for the feedback

**Review by principles31 — COMMENTED** [review]
# To keep your code clean, please don't leave commented code.
- The above changes has been implemented.

**Review by Wes-Isaac — APPROVED** [review]
Hi @principles31, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters are set up properly ✔️.
- Great summary😲. 
- Readme looks professional 👀.
- Project works as expected 😮

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principles31** [comment]
@Wes-Isaac Good work done kudus.
Please any words of motivation are welcome.

**Comment by Wes-Isaac** [comment]
### Your doing a great job keep rocking 🚀.

**Pull request merged**